apps/client/src/routes/dashboards/dashboards-index/dashboards-index-page.tsx

```diff
@@ -482,6 +475,31 @@ export function DashboardsIndexPage() {
               },
             },
           },
+          {
+            id: 'id',
+            header: intl.formatMessage({
+              defaultMessage: 'ID',
+              description: 'dashboards table ID column header',
+            }),
+            cell: (dashboard) => (
+              <Link
+                href={`/dashboards/${dashboard.id}`}
+                onFollow={(event) => {
+                  event.preventDefault();
```

Comment:
create a function named `handleOnFollow`  and then 
`onFollow={handleOnFollow}`

Reply:
done